python/hail/methods/statgen.py

```diff
@@ -1299,3 +1299,36 @@ def filter(self):
             Env.hail().methods.FilterAlleles.apply(
                 base._jvds, '({p})[aIndex - 1]'.format(p=filter_hql), row_hql, entry_hql, self._keep, self._left_aligned, self._keep_star))
         return cleanup(m)
+
+@handle_py4j
+@typecheck(left_aligned=bool)
+def min_rep(ds, left_aligned=False):
+    """Gives minimal, left-aligned representation of alleles. 
+
+    .. include:: ../_templates/req_tvariant.rst
+
+    Notes
+    -----
+    Note that this can change the variant position.
+
+    Examples
+    --------
+
+    Simple trimming of a multi-allelic site, no change in variant
+    position `1:10000:TAA:TAA,AA` => `1:10000:TA:T,A`
+
+    Trimming of a bi-allelic site leading to a change in position
+    `1:10000:AATAA,AAGAA` => `1:10002:T:G`
+
+    Parameters
+    ----------
+    left_aligned : bool
+        If ``True``, variants are assumed to be left aligned and have
+        unique loci.  This avoids a shuffle.  If the assumption is
+        violated, an error is generated.
+
+    Returns
+    -------
+    :class:`.MatrixTable`
+    """
+    return MatrixTable(ds._jvds.minRep(left_aligned))
```

Comment:
Are there any tests for this that need to be ported over?

Reply:
No :-/

Reply:
Can you add just the function call to the tests with a FIXME to make it an actual test with asserts?

Reply:
Done.

Reply:
oh I see you added it already.